src/core/types.ts

```diff
@@ -31,7 +33,7 @@ interface _CoreOptions<AvailableTranslations extends TranslationSourceRecord[] =
      */
     locale?:
         | (AvailableTranslations extends AvailableTranslations ? CreateLocalesUnionFromAvailableTranslations<AvailableTranslations> : never)
-        | (CustomTranslations extends CustomTranslations ? CreateLocalesUnionFromCustomTranslations<CustomTranslations> : never);
+        | (CustomTranslations extends CustomTranslations ? CreateLocalesUnionFromCustomTranslations<CustomTranslations> : SupportedLocales);
```

Comment:
Just wondering if this is sufficient to provide type inference for supported locales out-of-the-box.
```suggestion
        | (CustomTranslations extends CustomTranslations ? CreateLocalesUnionFromCustomTranslations<CustomTranslations> : never)
        | SupportedLocales;
```

Reply:
✅